src/go/rpk/pkg/cli/transform/logs.go

```diff
@@ -0,0 +1,470 @@
+/*
+* Copyright 2024 Redpanda Data, Inc.
+*
+* Use of this software is governed by the Business Source License
+* included in the file licenses/BSL.md
+*
+* As of the Change Date specified in that file, in accordance with
+* the Business Source License, use of this software will be governed
+* by the Apache License, Version 2.0
+ */
+
+package transform
+
+import (
+	"container/list"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"os/signal"
+	"regexp"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+	"github.com/twmb/franz-go/pkg/kgo"
+	"go.uber.org/zap"
+)
+
+var (
+	minTime = time.Unix(0, 0).UTC()
+	maxTime = time.Unix(math.MaxInt64, math.MaxInt64).UTC()
+)
+
+func newLogsCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var follow bool
+	var head, tail int
+	var since, until timequery
+	since.time = minTime
+	until.time = maxTime
+	cmd := &cobra.Command{
+		Use:     "logs [NAME]",
+		Aliases: []string{"log"},
+		Short:   "View logs for a transform",
```

Comment:
Should we add a `Long` help text? for example, we need to document the `--format wide/short` behavior

Reply:
Yes thanks for reminding me 😄